### ISSUE TITLE
Wagtail User Interface Accessibility Features Documentation

### ISSUE THREAD

**Issue by ArafatAbdussalam** [wagtail/guide#324]
**This issue is a WIP**

An Authoring tool is classified to be accessible only if it satisfies either level A, AA or AAA success criterion of the Authoring Tool Accessibility Guideline (ATAG 2.0). A required practice is that the features of the interface of the Authoring Tool is understandable to the authors who have accessibility needs and that has led me to participate in the Wagtail Accessibility Project. My interest in Accessibility aligns with the Wagtail Accessibility consideration for web content authors. Having gained experience using Wagtail as an editor and researched my findings by interacting with the interface, these are my suggested accessibility features documentation needs:

## Checklists
- [ ] Features Identification by identification number
- [ ] Identification of features not yet documented on the user guide
- [ ] Explanation of each feature functionality, reason the feature needs to be documented
- [ ] Suggested content demonstrating the documentation of each feature

**Comment by ArafatAbdussalam** [comment]
assign @ArafatAbdussalam

**Comment by ArafatAbdussalam** [comment]
Observation 1:
![Author](https://github.com/wagtail/guide/assets/92020171/a6e3637a-0d1d-4f17-95ce-88018f113405)

**Comment by ArafatAbdussalam** [comment]
My first authoring session was not quite smooth. One of the experience I had was the lack of understanding of the authors' checkbox. I did not understand how to add my name as an author. This is partly because it was not documented on the user guide.

My documentation suggestion:

**How-to Manage Authors**

Authors are individuals who create web contents. During the editing session, it is important that you include your name as an author. To do this, take the following steps:
- On the sidebar, navigate to the site data.

**Comment by thibaudcolas** [comment]
@ArafatAbdussalam please focus on more specific issues, this is way too broad as it is. There is also a fundamental need for you to understand Wagtail more before you report documentation gaps; both the things you pointed to aren’t Wagtail features per se (they’re site-specific customisations), so wouldn’t be documented as-is.

**Comment by ArafatAbdussalam** [comment]
Hi @thibaudcolas Thank you for the feedback. Initially, when I started this issue, I intended to make it be a summary of all issues I will be opening and append the links to the issues.

The comment below it was a draft. At the time of writing the comment, I was still familiarising myself with the admin interface and how to structure the documentation. 

I will appreciate your feedback on the issue I have just opened and documented better Issue #338
